plugins/outputs/dynatrace/dynatrace.go

```diff
@@ -28,7 +28,6 @@ var (
 	maxMetricKeyLen       = 250
 )
 
-var counts map[string]string
 var sent = 0
```

Comment:
You'll also want to move the sent variable inside the struct

Reply:
Added a variable SendCounter to the struct.